.dockerignore

```diff
@@ -0,0 +1,139 @@
+.env
+.venv
+
+**/__pycache__
+
+.dockerignore
+Dockerfile
+
+Vagrantfile
+
+# Created by https://www.gitignore.io/api/flask
+# Edit at https://www.gitignore.io/?templates=flask
+
+### Flask ###
+instance/*
+!instance/.gitignore
+.webassets-cache
+
+### Flask.Python Stack ###
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Scrapy stuff:
+.scrapy
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+# pyenv
+.python-version
+
+# celery beat schedule file
+celerybeat-schedule
+
+# SageMath parsed files
+*.sage.py
+
+# Spyder project settings
+.spyderproject
+.spyproject
+
+# Rope project settings
+.ropeproject
+
+# Mr Developer
+.mr.developer.cfg
+.project
+.pydevproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
+.pyre/
+
+# End of https://www.gitignore.io/api/flask
+
+## Intellij ###
+.idea/
+
+### macOS ###
+.DS_Store
+
+### Virtual environment ###
+env/
+.venv/
+
+### Configuration files ###
+.env
+
+### web drivers 
+geckodriver.*
+
+### Vagrant
+.vagrant
+
+### Logfiles
+*.log
```

Comment:
You're being specific with your Dockerfile's COPY commands so most of this file isn't really necessary but still certainly nice to have.

Looks like this is mostly your .gitignore. Some extra things you could include would be the `.git` folder or files your app doesn't need to run like README.md